common/test/acceptance/pages/lms/edxnotes.py

```diff
@@ -34,9 +34,9 @@ def _get_element_text(self, selector):
             return None
 
 
-class EdxNotesPageGroup(NoteChild):
+class EdxNotesChapterGroup(NoteChild):
     """
-    Helper class that works with note groups on Note page of the course.
+    Helper class that works with chapter (section) grouping of notes in the Course Structure view on the Note page.
```

Comment:
FYI, I just checked with Mark about this as there aren't clear guidelines anywhere. We do have a restriction of 80 characters in docstrings that get generated. For example, he changed some of ours in this PR:

https://github.com/edx/edx-platform/pull/7862/files

The biggest issue is around code samples as they can't be wrapped in HTML. Everything else is more of a style issue, but the recommendation is for consistency that docstrings should all be kept to 80 characters.